fcl/reco/MCC10/reco_uboone_data_mcc9_10.fcl

```diff
@@ -0,0 +1,981 @@
+
+microboone_reco_data_stage1:           [ rns,
+                                         @sequence::microboone_reco_data_optical,
+                                         @sequence::microboone_reco_data_signalprocessing
+                                       ]
+
+microboone_reco_data_stage1_double_pmt: [
+                                         rns,
+                                         @sequence::microboone_reco_data_optical_double_pmt,
+                                         @sequence::microboone_reco_data_signalprocessing
+                                       ]
+
+microboone_reco_data_mcc9_10_stage2:    [ rns,
+                                         opNoiseCreateMask,
+                                         wcopflash,
+                                         @sequence::microboone_reco_data_pandora_init,
+                                         @sequence::microboone_reco_data_nugraph2,
+                                         @sequence::microboone_reco_data_pandora,
+                                         @sequence::microboone_reco_data_crt,
+                                         blipreco
+                                       ]
+
+microboone_reco_data_mcc9_10_stage2_nocrt: [ rns,
+                                         opNoiseCreateMask,
+                                         wcopflash,
+                                         @sequence::microboone_reco_data_pandora_init,
+                                         @sequence::microboone_reco_data_nugraph2,
+                                         @sequence::microboone_reco_data_pandora,
+                                         blipreco
+                                       ]
+
+microboone_reco_data_mcc9_10_stage2_double_pmt: [
+                                         rns,
+                                         @sequence::microboone_reco_data_optical_double_pmt,
+                                         opNoiseCreateMask,
+                                         wcopflash,
+                                         @sequence::microboone_reco_data_pandora_init,
+                                         @sequence::microboone_reco_data_nugraph2,
+                                         @sequence::microboone_reco_data_pandora,
+                                         @sequence::microboone_reco_data_crt,
+                                         blipreco
+                                       ]
+
+microboone_reco_data_mcc9_10_stage2_reprocess: [
+                                         rns,
+                                         opNoiseCreateMask,
+                                         wcopflash,
+                                         @sequence::microboone_reco_data_nugraph2,
+                                         @sequence::microboone_reco_data_pandora,
+                                         @sequence::microboone_reco_data_crt,
+                                         blipreco
+                                       ]
```

Comment:
This is the "new" FCL sequence for the reco2 input files, to skip some parts of the processing.

Not sure if there is still too much there though, we may be able to drop a few other bits.